code/game/turfs/hybrisa.dm

```diff
@@ -0,0 +1,717 @@
+/turf/open/floor/hybrisa/engineership/engineer_floor7
+	icon_state = "engineer_floor_1"
+/turf/open/floor/hybrisa/engineership/engineer_floor8
+	icon_state = "engineer_floor_5"
+/turf/open/floor/hybrisa/engineership/engineer_floor9
+	icon_state = "engineer_metalfloor_4"
+/turf/open/floor/hybrisa/engineership/engineer_floor10
+	icon_state = "engineer_floor_corner1"
+/turf/open/floor/hybrisa/engineership/engineer_floor11
+	icon_state = "engineer_floor_corner2"
+/turf/open/floor/hybrisa/engineership/engineer_floor12
+	icon_state = "engineerwallfloor1"
+/turf/open/floor/hybrisa/engineership/engineer_floor13
+	icon_state = "outerhull_dir"
+/turf/open/floor/hybrisa/engineership/engineer_floor14
+	icon_state = "engineer_floor_corner3"
+
+// Pillars
+
+/turf/open/floor/hybrisa/engineership/pillars
+	name = "strange metal pillar"
+	desc = "A strange metal pillar, unlike any metal you've seen before."
+	icon_state = "eng_pillar1"
+
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar1
+	icon_state = "eng_pillar1"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar2
+	icon_state = "eng_pillar2"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar3
+	icon_state = "eng_pillar3"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar4
+	icon_state = "eng_pillar4"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth1
+	icon_state = "eng_pillarsouth1"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth2
+	icon_state = "eng_pillarsouth2"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth3
+	icon_state = "eng_pillarsouth3"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth4
+	icon_state = "eng_pillarsouth4"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest1
+	icon_state = "eng_pillarwest1"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest2
+	icon_state = "eng_pillarwest2"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest3
+	icon_state = "eng_pillarwest3"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest4
+	icon_state = "eng_pillarwest4"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast1
+	icon_state = "eng_pillareast1"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast2
+	icon_state = "eng_pillareast2"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast3
+	icon_state = "eng_pillareast3"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast4
+	icon_state = "eng_pillareast4"
+
+
+// -------------------- // Hybrisa Wall types // ---------------- //
+
+// Derelict Ship
+
+/turf/closed/wall/engineership
+	name = "strange metal wall"
+	desc = "Nigh indestructible walls that make up the hull of an unknown ancient ship."
+	icon = 'icons/turf/walls/engineership.dmi'
+	icon_state = "metal"
+	walltype = WALL_HUNTERSHIP
+	hull = TRUE
+
+/turf/closed/wall/engineership/destructible
+	name = "strange metal wall"
+	desc = "Nigh indestructible walls that make up the hull of an unknown ancient ship."
```

Comment:
Descriptions of these ought to vary if one is destructible and the other not.

Its also only necessary to change the vars when they actually change. E.g. icon only needs to be specified in `/turf/closed/wall/engineership` not in `/turf/closed/wall/engineership/destructible` Doesn't hurt, but it helps to understand what is changing with the subtype if only the changes are present.

Same applies for most of the following changes...